airflow/cli/cli_parser.py

```diff
@@ -36,13 +38,24 @@
 from airflow.exceptions import AirflowException
 from airflow.executors.executor_constants import CELERY_EXECUTOR, CELERY_KUBERNETES_EXECUTOR
 from airflow.executors.executor_loader import ExecutorLoader
+from airflow.models import XCOM_RETURN_KEY
```

Comment:
Are we fine with this dependency on the model layer here or it's better to refactor the constant out?

Reply:
Better to refactor the constant out.

Reply:
Created it as a separate PR to keep this one more focused (#30180).